bin/hocon

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'hocon'
+require 'hocon/config_factory'
+require 'hocon/parser/config_document_factory'
+
+# List of valid subcommands
+SUBCOMMANDS = ['get', 'set', 'unset']
+
+# Main entry point into the script
+# Calls the appropriate subcommand
+def main(opts)
+  case opts[:subcommand]
+    when 'get'
+      get_main(opts)
+    when 'set'
+      set_main(opts)
+    when 'unset'
+      unset_main(opts)
+  end
+
+  exit(0)
+end
+
+# Entry point for the 'get' subcommand
+# Prints the value at the path given on the command line
+def get_main(opts)
+  config = get_hocon_config(opts)
+
+  puts config.get_any_ref(opts[:path])
+end
+
+# Entry point for the 'set' subcommand
+# Prints the HOCON config after adding/replacing the value at the given path
+# with the given value
+def set_main(opts)
+  config_doc = get_hocon_doc(opts)
+  modified_config_doc = config_doc.set_value(opts[:path], opts[:new_value])
+
+  puts modified_config_doc.render
+end
+
+# Entry point for the 'unset' subcommand
+# Prints the HOCON config after removing the value at the given path
+def unset_main(opts)
+  config_doc = get_hocon_doc(opts)
+  modified_config_doc = config_doc.remove_value(opts[:path])
+
+  puts modified_config_doc.render
```

Comment:
It might make it easier to write tests for these if you move the `puts` calls up into the case statement in the `main` method, and just had these return  the values.  Then I think you could get like 90% of the test coverage you'll want, without having to shell out to call the actual CLI command.